app/views/products/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Vendor Porducts</h1>
+<ul>
+
+    <span><%=@products.name%></span>
+    <span><%=@products.vendor_id%></span>
+</ul>
```

Comment:
How are these displaying? I would think that `@products` would need to be looped through?